examples/ansible/code/Synchronize_DNS_Zones

```diff
@@ -0,0 +1,110 @@
+- name: Synchronize DNS
+  hosts: 24-bind-01
+  vars:
+    netbox_url: 'https://<netbox-url>/'
+    netbox_token: '<redacted>'
+    zone_file_path: '/path_to_your_zone_files/'
+  tasks:
+
+#set Variables
+    - name: get active Zone Names
+      ansible.builtin.set_fact:
+        zone_names: "{{ query('netbox.netbox.nb_lookup', 'zones', plugin='netbox_dns',
+                           api_endpoint=netbox_url,
+                           api_filter='active=true',
+                           token=netbox_token)
+                    | map(attribute='value') | map(attribute='name') }}"
+
+    - name: get used Views
+      ansible.builtin.set_fact:
+        used_views: "{{ query('netbox.netbox.nb_lookup', 'views', plugin='netbox_dns',
+                           api_endpoint=netbox_url,
+                           token=netbox_token)
+                    | map(attribute='value')| map(attribute='name') }}"
+
+#Create View-Folder and remove unwanted Files
+    - name: Create View-Directory
+      ansible.builtin.file:
+        path: "{{zone_file_path}}{{ item }}"
+        state: directory
+        mode: '755'
+      loop: "{{ used_views }}"
+
+    - name: Collect Directory contents
+      ansible.builtin.shell: >
+        ls -1 {{zone_file_path}}
+      register: directories
+      changed_when: False
+
+    - name: Remove unwanted Content
+      ansible.builtin.file:
+        path: "{{zone_file_path}}{{ item }}"
+        state: absent
+      with_items: "{{ directories.stdout_lines }}"
+      when: >
+        item not in used_views
+
+#Generate Zones for Views
+    - name: Generate Zone Files
+      ansible.builtin.template:
+          src: dns.zone.j2
```

Comment:
Umm ... where is the template? :-)

Reply:
Oh thats just the one provided, just happen i renamed it internally :D